src/main/java/com/mixpanel/android/viewcrawler/ViewVisitor.java

```diff
@@ -105,6 +109,64 @@ protected String name() {
         private final Object[] mOriginalValueHolder;
     }
 
+    public static class LayoutSetVisitor extends ViewVisitor {
```

Comment:
I'd like to change the "Set" in LayoutSetVisitor to maybe LayoutUpdateVisitor or LayoutRulesVisitor (or just LayoutVisitor)?